modules/jupyter-widget/src/version.js

```diff
@@ -0,0 +1,14 @@
+const data = require('../package.json');
```

Comment:
Nit: Importing `package.json` does bloat your module a little. But it is OK for now, it is robust (it is a little fiddly to use injected version strings).